run-wasm/src/main.rs

```diff
@@ -1,3 +1,3 @@
 fn main() {
-    cargo_run_wasm::run_wasm_with_css("body { margin: 0px; }");
+    cargo_run_wasm::run_wasm_with_css(include_str!("style.css"));
```

Comment:
Could probably just inline the `css` into this file, so it's a bit more easier to grasp.

Reply:
I un-inlined it on purpose so it can be properly formatted and color coded by an IDE, otherwise its just an ugly string.

Reply:
I'm unsure what the disadvantage is of a separate file here, so the syntax highlighting seems like a worthwhile advantage to me.

Reply:
That seems like a very fine trade-off to me.